custom-routes-vis/src/main/webapp/script.js

```diff
@@ -12,9 +12,138 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** Creates the world map. */
-function createWorldMap() {
+const LATLNG = {lat: 0.0, lng: 0.0};
+/**
+ * Marker names.
+ * @enum {string}
+ */
+const MarkerNames = {
+  ORIGIN: 'origin',
+  DESTINATION: 'destination',
+};
+// Array holding origin and destination markers.
+let originDestinationMarkers = []
+
+/** Creates the world map with markers. */
+function createWorldMapWithMarkers() {
   map = new google.maps.Map(
     document.getElementById('map'),
     {center: {lat: 38.46049, lng: -5.428423}, zoom: 3});
+  
+  // Create origin and destination markers, but hides them until user selects
+  // coordinates.
+  createMarker('origin-coordinates', 'A', 'Origin', LATLNG);
+  createMarker('destination-coordinates', 'B', 'Destination', LATLNG);
```

Comment:
Consider changing the title 'Origin' and 'Destination' to the constant MarkerNames as well

Reply:
The enum MarkerNames are all lowercase so this is different. I am talking to Haylee about how to address this.

Reply:
Got it! This is good as is